app/components/Header.jsx

```diff
@@ -27,7 +27,7 @@ export default class Header extends PureComponent {
 
   render() {
     return (
-      <div className="row row--stacked">
+      <div className="header__wrap">
```

Comment:
Just checked it out on safari. Just remove the class entirely and add a margin of `$block-margins` to the search div and it should be good